### PR TITLE
Support building base-runner on aarch64.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Related: #8164

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
/gcbrun trial_build.py skcms --sanitizer address coverage memory undefined --fuzzing-engine libfuzzer afl honggfuzz

File: infra/base-images/base-runner/install_java.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# Install java in a platform-aware way.
+
+ARCHITECTURE=
+case $(uname -m) in
+    x86_64)
+      ARCHITECTURE=x64
+        ;;
+    aarch64)
+      ARCHITECTURE=aarch64
```
Comment (oliverchang) [inline infra/base-images/base-runner/install_java.sh:26]: We probably don't care too much for aarch64 for JAva either?

File: infra/base-images/base-runner/Dockerfile [hunk 2]
```diff
@@ -35,21 +35,8 @@ COPY --from=base-clang /usr/local/bin/llvm-cov \
...
-    libc6-i386 \
-    libcap2 \
-    python3 \
-    python3-pip \
-    python3-setuptools \
-    rsync \
-    unzip \
-    wget \
-    zip --no-install-recommends
+COPY install_deps.sh /
```
Comment (oliverchang) [inline infra/base-images/base-runner/Dockerfile:38]: Is this file missing from the PR?

File: infra/base-images/base-runner/install_java.sh (hunk 1 above)
Comment (jonathanmetzman) [reply infra/base-images/base-runner/install_java.sh:26]: Indeed we don't. I figure though it's better to make the images the same for now and we can worry about optimizing later.

File: infra/base-images/base-runner/Dockerfile (hunk 2 above)
Comment (jonathanmetzman) [reply infra/base-images/base-runner/Dockerfile:38]: Yes. Added.

**Comment by jonathanmetzman** [comment]
/gcbrun trial_build.py skcms --sanitizer address coverage memory undefined --fuzzing-engine libfuzzer afl honggfuzz

**Review by oliverchang — APPROVED** [review]

**Comment by jonathanmetzman** [comment]
/gcbrun trial_build.py all --sanitizer address coverage --fuzzing-engine libfuzzer

**Comment by jonathanmetzman** [comment]
/gcbrun trial_build.py all --sanitizer address coverage --fuzzing-engine libfuzzer

**Review by oliverchang — APPROVED** [review]

**Pull request merged**